platform-sdk/swirlds-component-framework/src/main/java/com/swirlds/component/framework/schedulers/internal/ConcurrentTask.java

```diff
@@ -61,14 +61,22 @@ class ConcurrentTask extends AbstractTask {
      * {@inheritDoc}
      */
     @Override
-    protected boolean exec() {
+    protected boolean execImpl() {
         try {
             handler.accept(data);
         } catch (final Throwable t) {
-            uncaughtExceptionHandler.uncaughtException(Thread.currentThread(), t);
+            completeExceptionallyImpl(t);
```

Comment:
Repeating my previous question: why do we call `completeExceptionallyImpl()` here and `completeExceptionally()` in `SequentialTask`? Which is correct? (I guess `completeExceptionally()`).

Reply:
Oh, this is definitely a bug. Should be `completeExceptionally()` in both places. Let me check other tasks, too

Reply:
Fixed